plugin.go

```diff
@@ -146,8 +147,14 @@ func validateCommand() *exec.Cmd {
 	)
 }
 
+func initCommand() * exec.Cmd {
+   return exec.Command("terraform", "init",)
+}
+
+
 func planCommand(variables map[string]string, secrets map[string]string, parallelism int, targets []string) *exec.Cmd {
-	args := []string{
+	exportSecrets(secrets)
```

Comment:
`exportSecrets` is already called earlier @ https://github.com/josmo/drone-terraform/blob/1847409c8d1cb007712b217181f1031e788beb28/plugin.go#L49